### ISSUE TITLE
Add release tarball to the download page

### ISSUE THREAD

**Issue by Fat-Zer** [euroelessar/qutim#335]
Congratulations with the new release of the 0.3 branch!

Can you please add it to the download page of the site: http://www.qutim.org/download

**Comment by Fat-Zer** [comment]
It's been more than half a year since the issue was filled in... I believe it's good time to memorise about it yet another time.

**Comment by Fat-Zer** [comment]
Could you package a normal tarball, please?
I need it for bring the 0.3.3 version to gentoo... 
Note that github-provided tarball lacks of submodules, and the link to core/3rdparty/k8json seems to be lost at all...

**Comment by nico-izo** [comment]
@Fat-Zer Here you are:
https://qutim.org/file/78

It can also be downloaded from https://qutim.org/download from tab "Source".

**Comment by Fat-Zer** [comment]
thanks

**Comment by Fat-Zer** [comment]
Oh, wait, you seems to be packaged wrong protocols/vkontakte/vreen version...

**Comment by nico-izo** [comment]
Ahem, I just used 
git checkout qt4 && git submodule update --init --recursive

There is broken k8json dependency, so I locally changed .gitmodules to fetch all sources, but it is the only change.

I can't see any differences in pair vreen/qt4-vreen.

**Comment by nico-izo** [comment]
@Fat-Zer Okay, I see.
It seems that there is old vreen with broken Qt preprocessor version check. I fixed it and uploaded new tarball here: https://qutim.org/file/79
Diff: http://paste.ubuntu.com/12136947/

**Comment by Fat-Zer** [comment]
> Ahem, I just used
git checkout qt4 && git submodule update --init --recursive

Ohmm... I believe you were supposed to checkout v0.3.3 tag rather than the branch itself in its current state... 

Sorry for bothering you, but I'd love to see the release tarball following the git tags...
Although if you think the recent changes are critical, please name the tarball so it could been recognized differently e.g. 0.3.3_p1...

**Comment by nico-izo** [comment]
I think, branch in 0.3.3 state does not have at least two small but needful fixed backported by me from 0.4 branch.
Also, I just cannot checkout 0.3.3 tag without changes due to dead submodules.

So... The exact copy of previous archive, but named 0.3.3.p_1
https://qutim.org/file/80

And 0.3.3 tag, as you wished (there are three dead submodules in 0.3.3 tag, so if you want to checkout by yourself, use submodules from this commit: d5653e27b62cf99e1591253638bfcd07dd211710).
https://qutim.org/file/81

**Comment by Fat-Zer** [comment]
>Also, I just cannot checkout 0.3.3 tag without changes due to dead submodules.

If you wonder, this is how I've managed to do it (copy-paste version): 
```
git clone https://github.com/euroelessar/qutim.git
cd qutim
git checkout qt4
git submodule update --init
pushd protocols/vkontakte/vreen/
sed -i 's!https://gitorious.org/k8jsonqt/qutim-developers-k8json.git!git://github.com/euroelessar/k8json.git!' .gitmodules
git submodule update --init --recursive
git checkout HEAD -- .gitmodules
popd
git submodule update --init --recursive
git checkout v0.3.3
git submodule update --init --recursive
```

>So... The exact copy of previous archive, but named 0.3.3.p_1

ok, thanks, I'll use this one...
